src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -900,6 +901,11 @@ public static void firePlayerSmeltedEvent(Player player, ItemStack smelted)
         MinecraftForge.EVENT_BUS.post(new PlayerEvent.ItemSmeltedEvent(player, smelted));
     }
 
+    public static void fireClientPauseEvent(boolean paused)
+    {
+        MinecraftForge.EVENT_BUS.post(new ClientPauseEvent(paused));
+    }
+
```

Comment:
Please move this event to `net.minecraftforge.client.ForgeHooksClient`, as this class only contains common event factories and firing.

Reply:
done